sydent/db/hashing_metadata.py

```diff
@@ -52,6 +57,8 @@ def get_lookup_pepper(self) -> Optional[str]:
         if isinstance(pepper, bytes):
             pepper = pepper.decode("UTF-8")
 
+        self._cached_lookup_pepper = pepper
+
         return pepper
 
     def store_lookup_pepper(
```

Comment:
Should this function not invalidate the cache?

Reply:
i.e. `store_lookup_pepper`. (I don't think github has helped express my intent here)